doc/reference/modules/basic_mapping.xml

```diff
@@ -2656,9 +2673,14 @@
                         <row>
                             <entry><literal>DbTimestamp</literal></entry>
                             <entry><literal>System.DateTime</literal></entry>
-                            <entry><literal>DbType.DateTime</literal> - as specific as database supports.</entry>
-                            <entry><literal>type="DbTimestamp"</literal> must be specified.  When used as a <literal>version</literal>
-                            field, uses the database's current time rather than the client's current time.</entry>
+                            <entry>
+                                <literal>DbType.DateTime</literal> / <literal>DbType.DateTime2</literal><coref linkend="basic_mapping.datetime-co" />
+                            </entry>
+                            <entry>
+                                <literal>type="DbTimestamp"</literal> must be specified.  When used as a
+                                <literal>version</literal> field, uses the database's current time retrieved
+                                in dedicated queries, rather than the client's current time.
```

Comment:
Further hinting at the performance consequences it may have by mentioning it does that in dedicated queries.